.doc_gen/metadata/s3-control_metadata.yaml

```diff
@@ -132,5 +132,24 @@ s3-control_Basics:
             - description: An action class that wraps operations.
               snippet_tags:
                 - s3control.java2.job.actions.main
+    Python:
```

Comment:
This scenario includes some additional actions such as tagging, listing, and updating the priority - if you want to add it here, it should include that functionality as well. I believe the Java example has those functions, and you can see the [full specification here](https://github.com/awsdocs/aws-doc-sdk-examples/blob/67e73648da4abf0c0d01c37c190f7cbf04b8cc31/scenarios/basics/s3/SPECIFICATION.md). You'll also want to add snippet tags around the individual action code so that it can be shown in the actions blocks above.

Reply:
Added snippet tags for each operations involved